app/models/sms/adapters/adapter.rb

```diff
@@ -79,6 +79,7 @@ def send_request(uri, method = :get, payload = {})
     when :get
       request = Net::HTTP::Get.new(uri.request_uri)
     when :post # only used for FrontlineCloud
+      http.use_ssl = true
```

Comment:
I figured maybe we could cross that bridge when we came to it. It's also assuming a JSON payload because I didn't want to get too tangled trying to generalize it for currently non-existent future cases.